components/chef-ui-library/src/atoms/chef-input/chef-input.tsx

```diff
@@ -95,7 +97,8 @@ export class ChefInput {
         placeholder={ this.placeholder }
         onChange={ this.handleChange.bind(this) }
         disabled={this.disabled}
-        aria-disabled={this.disabled}></input>
+        aria-disabled={this.disabled}
+        autocomplete="off"/>
```

Comment:
It is a logical assumption to assume that `<foo></foo>` and `<foo/>` are equivalent. I know that in earlier HTML versions, some elements did not behave quite correctly if they were self-closing. From what I've read on HTML 5, though, that seems to no longer be an issue, so making this self-closing should work OK. Can anyone else confirm?

Reply:
We've had self-closing inputs in `automate-ui` in a few places, so I've figured this would be OK.

Reply:
Elements like `input`, `img`, `link`, etc have always been fine to be self-closing, even before html5. However I believe html5 made the forward slash optional: `<img>` vs `<img />`.

Reply:
I'd add `@Prop() autocomplete = 'off'` so that the value can be configured from the user of `chef-input`.

```
@Prop() autocomplete = 'off';
       
    ...
    aria-disabled={this.disabled}
    autocomplete={this.autocomplete} />
```

Reply:
✔️ done